linera-service/src/grpc_proxy.rs

```diff
@@ -264,7 +279,12 @@ impl ValidatorNode for GrpcProxy {
                     .inc();
                 Ok(chain_info_result)
             }
-            Err(status) => Err(status),
+            Err(status) => {
+                PROXY_REQUEST_ERROR
+                    .with_label_values(&["handle_lite_certificate"])
+                    .inc();
+                Err(status)
+            }
```

Comment:
<i class='graphite__hidden'>[Re: lines 276 to 287]<br/></i>

Can we deduplicate this, so we just call:

```rust
Self::handle_proxy_request_outcome(result, "handle_lite_certificate")
```<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1172?utm_source=unchanged-line-comment">Graphite</a>.</span>